serve.go

```diff
@@ -501,6 +506,15 @@ type config struct {
 	err error
 }
 
+func (c *config) setupCommandCallback() {
```

Comment:
Not everything on the queue is readily marshal-able to JSON. Most of the pubsub messages are protobufs and I specifically wanted to keep that away from (this particular) callback. I.e. submit command w/ JSON API, receive URL callback of JSON.